### ISSUE TITLE
Integrate phylogenetic estimates with phenoscape

### ISSUE THREAD

**Issue by snacktavish** [phenoscape/KB-DataFest-2017#12]
Currently the taxon browser of the KB returns a newick tree based on the vertebrate taxonomy ontology.
( e.g.  [link](http://kb.phenoscape.org/#/taxon/http://purl.obolibrary.org/obo/VTO_9016144?phenotypes.quality_type=quality-phenotype&phenotypes.entity_parts=false&phenotypes.entity_historical_homologs=false&phenotypes.entity_serial_homologs=false&tab=properties) )

Calling [OpenTree](https://tree.opentreeoflife.org) API’s to get subtrees instead or in addition would provide interconnection with up to date phylogentic inferences, as well as taxonomy.

**Comment by wdahdul** [comment]
Very interested in this. One issue that Phenoscape has been discussing is that Phenoscape's taxonomy ontology (VTO) is partly based on Catalog of Fishes, which Open Tree doesn't currently reference. As a result, many taxa with phenotypes in Phenoscape are missing from the OpenTree taxonomy.

**Comment by balhoff** [comment]
@snacktavish @mtholder we would like to transition from VTO to a taxonomy based on Open Tree. For us it is important that Catalog of Fishes be added to the taxonomy merge for OTT. It would be great to discuss this while you're in town. If you're amenable to adding it, we can help with getting access to structured data, which unfortunately isn't as straightforward as one would hope.

**Comment by jar398** [comment]
Known open tree problem: https://github.com/OpenTreeOfLife/reference-taxonomy/issues/172

It wouldn't be very hard to import fishbase to open tree, but somebody needs to take charge of it.

Fishbase will be clearly superior at the species and genus level, but has anyone compared higher taxonomy of fishes between fishbase and NCBI? In some groups, NCBI is quite progressive.

**Comment by balhoff** [comment]
@wdahdul @pmabee do you know how the Fishbase taxonomy relates to Catalog of Fishes? Would going through Fishbase be a way to have an up-to-date Catalog of Fishes taxonomy?

**Comment by wdahdul** [comment]
Catalog of Fishes is preferable. I recall that while Fishbase does reference Catalog of Fishes, it also contains names that may not be recognized as valid by CoF (see 
 http://www.fishbase.org/Nomenclature/FBCofFNames.php), and we want to contain valid names. 
Also, CoF adds newly described taxa very quickly, and seems that Fishbase also doesn't update to CoF often -- for example Danio htamanthinus described in 2016 is missing from Fishbase but in CoF.

**Comment by laurajackson** [comment]
Yes, FishBase incorporates CoF although uses an outdated 2014 version. Since FishBase reports more than just taxonomic information, it doesn’t frequently get updated with newly described species that would be in CoF causing differences in species count (e.g., teleost species in FishBase: 32,1216 and current CoF: 33,191). But, currently this is the only way you can get a list of CoF species, without being able to get it directly from the CoF database. They don’t have an option to download, and I also contacted Bill Eschmeyer (curator of FishBase), who told me that I could not get a list of species. Therefore, FishBase would be the only viable proxy for species from CoF, unless the downloadable option is implemented in CoF.
